packages/eas-cli/src/credentials/manager/ManageIos.ts

```diff
@@ -32,66 +33,76 @@ enum ActionType {
-          },
-          {
-            value: ActionType.UseExistingDistributionCertificate,
-            title: 'Use existing Distribution Certificate in current project',
-          },
-          {
-            value: ActionType.RemoveSpecificProvisioningProfile,
-            title: 'Remove Provisioning Profile from current project',
-          },
-        ]
-      : [];
+        const projectSpecificActions: { value: ActionType; title: string }[] = ctx.hasProjectContext
+          ? [
+              {
+                // This command will be triggered during build to ensure all credentials are ready
+                // I'm leaving it here for now to simplify testing
+                value: ActionType.SetupBuildCredentials,
+                title: 'Ensure all credentials for project are valid',
+              },
+              {
+                value: ActionType.UpdateCredentialsJson,
+                title: 'Update credentials.json with values from Expo servers',
+              },
+              {
+                value: ActionType.SetupBuildCredentialsFromCredentialsJson,
+                title: 'Update credentials on Expo servers with values from credentials.json',
+              },
+              {
+                value: ActionType.UseExistingDistributionCertificate,
+                title: 'Use existing Distribution Certificate in current project',
+              },
+              {
+                value: ActionType.RemoveSpecificProvisioningProfile,
+                title: 'Remove Provisioning Profile from current project',
+              },
+            ]
+          : [];
 
-    const { action } = await promptAsync({
-      type: 'select',
-      name: 'action',
-      message: 'What do you want to do?',
-      choices: [
-        ...projectSpecificActions,
-        {
-          value: ActionType.CreateDistributionCertificate,
-          title: 'Add new Distribution Certificate',
-        },
-        {
-          value: ActionType.RemoveDistributionCertificate,
-          title: 'Remove Distribution Certificate',
-        },
-        {
-          value: ActionType.UpdateDistributionCertificate,
-          title: 'Update Distribution Certificate',
-        },
-      ],
-    });
+        const { action } = await promptAsync({
+          type: 'select',
+          name: 'action',
+          message: 'What do you want to do?',
+          choices: [
+            ...projectSpecificActions,
+            {
+              value: ActionType.CreateDistributionCertificate,
+              title: 'Add new Distribution Certificate',
+            },
+            {
+              value: ActionType.RemoveDistributionCertificate,
+              title: 'Remove Distribution Certificate',
+            },
+            {
+              value: ActionType.UpdateDistributionCertificate,
+              title: 'Update Distribution Certificate',
+            },
+          ],
```

Comment:
Maybe an exit action would be useful? I assume the only way of exiting the manager is by pressing `CTRL + C`?